articles/virtual-machines/workloads/sap/azure-monitor-sap-quickstart.md

```diff
@@ -78,13 +78,29 @@ Sign in to the Azure portal at https://portal.azure.com
 
 1. Select OS (Linux) from the drop-down 
 
-> [!IMPORTANT]
-> To configure OS (Linux) provider, ensure that Node_Exporter is installed in each BareMetal instance. For more information, see [Node_Exporter](https://github.com/prometheus/node_exporter)
```

Comment:
Please include Baremetal/Virtual Machine. Since OS (& AMS) is agnostic to underlying hardware. Also, is Baremetal official term? for SAP bare metal, official term is Large Instance.